kolibri/plugins/user_profile/assets/src/composables/useIndividualDevice.js

```diff
@@ -9,6 +9,7 @@ export default function useIndividualDevice() {
     client({ url: urls['kolibri:kolibri.plugins.user_profile:userindividual']() }).then(
       response => {
         isIndividual.value = response.data.individual && !response.data.lod;
+        // isIndividual.value = true;
```

Comment:
ah, no, I use that code to test in my laptop. I have more than one user in my facility and commenting it out I will see the option to change facility. I left it there on purpose in case someone else developing this part is in the same situation.
`isIndividual` is used to determine if the option to change facility is shown in the profile screen.

For production that can be deleted, that `isIndividual` property must be active in devices with only one user and not being a LOD.